pkg/rtcmos/rtcmos_test.go

```diff
@@ -14,107 +14,103 @@ const (
 func TestSpatialLayer(t *testing.T) {
 	{
 		stat := Stat{
-			Bitrate: 620000,
-			VideoConfig: &VideoConfig{Width: int32Ptr(640), Height: int32Ptr(360), FrameRate: int32Ptr(15),
-				ExpectedWidth: int32Ptr(1280), ExpectedHeight: int32Ptr(720)},
```

Comment:
Removing the expected width/height and scaling the score based on difference and leaving the model as per the  JS repo. I think there are issues with that model too. Will elaborate on that later.

Any layer mismatch between expected vs available is handled in LK server by penalising one level for a missed layer. Will be in the LK server PR.